contracts/citizens/DeveloperSkillLevels.sol

```diff
@@ -30,16 +32,20 @@ contract DeveloperSkillLevels {
     mapping(address => uint256) private skillLevelRatingsSum;
     mapping(address => mapping(address => uint8)) public skillLevelRatings;
     IPassportUtils public passportUtils;
+    IEAS private immutable eas;
+    bytes32 private immutable easSchemaUID;
 
     error NotPassportOwner(address illegalAlien);
     error PassportExpired(address citizen);
     error RatingValueError(uint8 rating);
 
     event Rated(address developer, uint8 rating, address citizen);
 
-    constructor(address passportUtils_) {
+    constructor(address passportUtils_, address eas_, bytes32 easSchemaUID_) {
         owner = address(msg.sender);
         passportUtils = IPassportUtils(passportUtils_);
+        eas = IEAS(eas_);
+        easSchemaUID = easSchemaUID_;
```

Comment:
What schema are we going to use here?

Reply:
@johnmark13 We will have to create the schema on each chain we deploy our contracts to.  For now, Sepolia and Mainnet.

The Sepolia schema is here:  https://sepolia.easscan.org/schema/view/0x8233d9319f24851e27b79cd7c3afe2e22a125b722435733d8b428b85d6e2ab8b

It has two fields:  `string role,uint8 rating`

So we can use this one schema for multiple roles, and we can add new roles in the future if we want.